### PR TITLE
Interval can now be used to configure the advertised time resolution for liveupdate

### PR DESCRIPTION
**Pull request by maartenplieger**
See doc/tutorials/Configure_solar_terminator.md for details

### CODE REVIEW

File: adagucserverEC/LayerTypeLiveUpdate/LayerTypeLiveUpdate.cpp [hunk 1]
```diff
@@ -105,16 +105,24 @@ int layerTypeLiveUpdateConfigureWMSLayerForGetCapabilities(MetadataLayer *metada
     metadataLayer->layerMetadata.title.copy(metadataLayer->dataSource->cfgLayer->Name[0]->value.c_str());
   }
   CTime timeInstance;
+
+  CT::string timeResolution = "PT10M";
```
Comment (belentorrente) [inline adagucserverEC/LayerTypeLiveUpdate/LayerTypeLiveUpdate.cpp:109]: Maybe mention somewhere in the documentation that this is the default time resolution

File: data/config/datasets/adaguc.tests.solarterminator.xml [hunk 2]
```diff
@@ -22,6 +22,7 @@
     <DataPostProc algorithm="solarterminator"/>
     <Variable>solarterminator</Variable>
     <Styles>soltstyle</Styles>
+    <Dimension interval="PT1M">time</Dimension>
```
Comment (belentorrente) [inline data/config/datasets/adaguc.tests.solarterminator.xml]: I would put 10M here (1M looks like overkill for something like the solar terminator)

**Review by belentorrente — DISMISSED** [review]

File: adagucserverEC/LayerTypeLiveUpdate/LayerTypeLiveUpdate.cpp (hunk 1 above)
Comment (maartenplieger) [reply adagucserverEC/LayerTypeLiveUpdate/LayerTypeLiveUpdate.cpp:109]: Wrote a comment in doc/configuration/Layer.md

File: data/config/datasets/adaguc.tests.solarterminator.xml (hunk 2 above)
Comment (maartenplieger) [reply data/config/datasets/adaguc.tests.solarterminator.xml]: Updated!

**Review by belentorrente — APPROVED** [review]

**Pull request merged**